jsonpickle/unpickler.py

```diff
@@ -292,6 +296,28 @@ def loadrepr(reprstr):
     return eval(evalstr, mylocals)
 
 
+def loadmodule(module_str):
+    """Returns a reference to a module.
+
+    >>> obj = loadmodule('datetime/datetime.datetime')
+    >>> obj.__name__
+    'datetime'
+
+    """
+    module, evalstr = module_str.split('/')
+    mylocals = locals()
+    localname = module
+    if '.' in localname:
```

Comment:
I don't recall the exact semantics of loading a module, but is it possible for there to be 2+ dots, like if someone tries to import something like a function inside a class inside a file (for example, datetime/datetime.datetime.fromtimestamp)? This might not even be possible in python though, I'm unclear. If it is possible, we should add a test for that case.